python/tk_multi_workfiles/util.py

```diff
@@ -89,6 +89,20 @@ def value_to_str(value):
         # For everything else, just return as string
         return str(value)
 
+def get_sg_entity_name_field(entity_type):
+    """
+    :returns: A string, the field holding the Entity name for a given Entity type.
+    """
+    # Deal with some known special cases and assume "code" for anything else.
+    return {
+        "Project": "name",
```

Comment:
Why are we not using the code you introduced in core recently? Is it simply because we want to avoid requiring a really recent version of tk-core? This would be nice to point out, with the date the feature was introduced in core, so someone who sees the code at a later date can decide if the time has come to use the new core version.

Reply:
yep, adding a comment.